diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/BlockStructureBraces.kt

```diff
@@ -232,7 +233,12 @@ class BlockStructureBraces(configRules: List<RulesConfig>) : DiktatRule(
         if (!configuration.closeBrace) {
             return
         }
-        val space = node.findChildByType(RBRACE)!!.treePrev
+        var space = node.findChildByType(RBRACE)!!.treePrev
+        node.findParentNodeWithSpecificType(ElementType.LAMBDA_ARGUMENT)?.let {
+            if (space.text.isEmpty()) {
+                space = space.treePrev
```

Comment:
This line is marked as not covered with tests. Which case does this branch cover? Is it present in tests?
(Maybe this rule is activated after BlankLines rule? Then this check might be redundant)

Reply:
This rule always worked for lambda with empty BLOCK. Since the BLOCK node was checked, but not checked WHITE_SPACE

Reply:
I will add a test for this bug